Pong.java

```diff
@@ -97,17 +193,20 @@ private void movePlayer (Player player, int destination) {
 				player.position = getHeight() - HEIGHT;
 		}
 	}
-	
-	// Compute player position
-	private void computePosition (Player player) {
-		// MOUSE
+
+    /**
+     * Computes the destination of the paddle for the next frame depending on which type of player is moving the
+     * current paddle.
+     * @param player
+     */private void computePosition (Player player) {
+		//If the player type is mouse, move the paddle towards the mouse cursor.
```

Comment:
Add a space at the beginning of the comment. (Applies also to several other comments below).